src/writers/writer.ts

```diff
@@ -16,6 +16,7 @@ export interface IWriteOptions {
   xAlign: string;
   yAlign: string;
   textRotation: number;
+  textShear?: number;
```

Comment:
I think leaving the textrotation as it is makes sense. Text rotation will safely rotate your text in the cardinal directions and stay within the bounding box. text shear has to make some simplifying assumptions to keep the calculations easy. as a result it breaks out of the bounding box. I want to keep that behavior separated from the "safe" textrotation